src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionPatchXpathFilterTest.java

```diff
@@ -115,4 +106,28 @@ public void testMemberName() throws Exception {
         testByConfig(
                 "MemberName/patchedline/defaultContextConfig.xml");
     }
+
+    @Test
+    public void testAnonInnerLength() throws Exception {
+        testByConfig(
+                "AnonInnerLength/newline/defaultContextConfig.xml");
```

Comment:
You're adding more tests for newline and patchedline strategies. But in this issue you should add tests for context strategy.
You can keep these tests though, since you already created them.

Reply:
done